hashbang/welcome.post

```diff
@@ -0,0 +1,8 @@
+   * To learn more about us and our offerings type: man hashbang
```

Comment:
Mmmh, I disagree on that one: currently, all files start with a non-empty line, and I think it's easier to get it right if it's consistent.